src/main/java/uk/gov/companieshouse/exemptions/util/DateUtils.java

```diff
@@ -1,13 +1,24 @@
 package uk.gov.companieshouse.exemptions.util;
 
+import static java.time.ZoneOffset.UTC;
+
 import java.time.Instant;
-import java.time.ZoneOffset;
+import java.time.OffsetDateTime;
 import java.time.format.DateTimeFormatter;
+import org.apache.commons.lang.StringUtils;
 
 public class DateUtils {
```

Comment:
Can you add a private constructor to prevent instantiation. Like this:
`private DateUtils() {}`